packages/application-shell/src/components/navbar/navbar.js

```diff
@@ -130,28 +134,48 @@ MenuExpander.propTypes = {
   onClick: PropTypes.func.isRequired,
 };
 
-export const MenuGroup = props => (
-  <ul
-    className={classnames(
-      { [styles.list]: props.level === 1 },
-      { [styles.sublist]: props.level === 2 },
-      {
-        [styles['sublist-no-children']]: props.level === 2 && !props.children,
-      },
-      {
-        [styles['sublist-expanded__active']]:
-          props.level === 2 && props.isActive && props.isExpanded,
-      },
-      props.level === 2 && props.isActive && !props.isExpanded
-        ? styles['sublist-collapsed__active']
-        : styles.sublist__inactive
-    )}
-  >
-    {props.children}
-  </ul>
-);
+export const MenuGroup = props => {
+  const isSublistActiveWhileIsMenuExpanded =
+    props.level === 2 && props.isActive && props.isExpanded;
+  const isSublistActiveWhileIsMenuCollapsed =
+    props.level === 2 && props.isActive && !props.isExpanded;
+  return (
+    <ul
+      id={`${props.id}-group`}
+      data-testid={`${props.id}-group`}
+      role="menu"
+      aria-expanded={
+        isSublistActiveWhileIsMenuExpanded ||
+        isSublistActiveWhileIsMenuCollapsed
+      }
+      className={classnames(
+        { [styles.list]: props.level === 1 },
+        { [styles.sublist]: props.level === 2 },
+        {
+          [styles['sublist-no-children']]: props.level === 2 && !props.children,
+        },
+        {
+          [styles[
+            'sublist-expanded__active'
+          ]]: isSublistActiveWhileIsMenuExpanded,
+        },
+        {
+          [styles[
+            'sublist-collapsed__active'
```

Comment:
nit; what if we just remove the dash so this can be `styles.sublistCollapsed__active`?

sidenote: I'm not sure if I like BEM approach when using CSSmodules anymore

Reply:
Yeah, like I said above the navbar component needs to be rewritte, so we can get rid of all this stuff once we do that.